unlock-app/src/components/content/EventContent.tsx

```diff
@@ -9,6 +9,8 @@ import EventDetails from './event/EventDetails'
 import { EventLandingPage } from './event/EventLandingPage'
 import { useRouterQueryForLockAddressAndNetworks } from '~/hooks/useRouterQueryForLockAddressAndNetworks'
 import { useMetadata } from '~/hooks/metadata'
+import { toFormData } from '~/components/interface/locks/metadata/utils'
+import { Event, PaywallConfigType } from '@unlock-protocol/core'
 
 export const EventContent = () => {
   const {
```

Comment:
This is the legacy path (before slugs existed on locks...) We have ti reconstruct the whole object from the lock address.